teams_messages.c

```diff
@@ -2023,6 +2019,19 @@ teams_set_status(PurpleAccount *account, PurpleStatus *status)
 	teams_set_mood_message(sa, purple_status_get_attr_string(status, "message"));
 }
 
+
+gboolean
+teams_set_status_timeout_cb(TeamsAccount* sa)
+{
+    if(sa == NULL || sa->account == NULL) {

```

Comment:
Sadly, `sa` will never be `NULL` here, as glib just passes along whatever memory pointer it was given during the `g_timeout_add_seconds()` call, but there's no harm having this in here :)